src/com/kroo/typeset/logback/JsonLayout.clj

```diff
@@ -233,6 +245,10 @@
   (vswap! (.state this) update-opt+mapper
           :order-by-keys sort-keys?))
 
+(defn -setJacksonModules [this modules]
+  (vswap! (.state this) update-opt+mapper
+          :jackson-modules modules))
```

Comment:
Yeah I did wonder that initially, and now thinking about it, it means you only create them once when they are being set rather than every time an object-mapper property changes (since we re-generate the object-mapper), so I think that's a good idea. It also means we can just store the `modules` property straight into the opts.